pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -573,16 +573,15 @@ func (b *volumeBinder) bindAPIUpdate(ctx context.Context, pod *v1.Pod, bindings
 	// Do the actual prebinding. Let the PV controller take care of the rest
 	// There is no API rollback if the actual binding fails
 	for _, binding = range bindings {
-		klog.V(5).InfoS("bindAPIUpdate: binding PV to PVC", "pod", klog.KObj(pod), "PV", klog.KObj(binding.pv), "PVC", klog.KObj(binding.pvc))
 		// TODO: does it hurt if we make an api call and nothing needs to be updated?
-		klog.V(2).InfoS("Claim bound to volume", "PVC", klog.KObj(binding.pvc), "PV", klog.KObj(binding.pv))
+		klog.V(2).InfoS("Updating PersistentVolume: binding to claim", "pod", klog.KObj(pod), "PV", klog.KObj(binding.pv), "PVC", klog.KObj(binding.pvc))
 		newPV, err := b.kubeClient.CoreV1().PersistentVolumes().Update(ctx, binding.pv, metav1.UpdateOptions{})
 		if err != nil {
-			klog.V(4).InfoS("Updating PersistentVolume: binding to claim failed", "PV", klog.KObj(binding.pv), "PVC", klog.KObj(binding.pvc), "err", err)
+			klog.V(4).InfoS("Updating PersistentVolume: binding to claim failed", "pod", klog.KObj(pod), "PV", klog.KObj(binding.pv), "PVC", klog.KObj(binding.pvc), "err", err)
```

Comment:
Is this just a V(4) because the error is logged higher in the call stack as well?

Reply:
Can you verify what happens with the returned error?

Reply:
According to the code, if an error occurs it will be returned to retry scheduling. It goes back to `handleBindingCycleError` in schedule_one.go.

https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/volumebinding/volume_binding.go#L322-L349

Reply:
Right, so ultimately it would be logged here https://github.com/kubernetes/kubernetes/blob/a5f59ccf37fabacf0339aeaf95b6929db7460f64/pkg/scheduler/schedule_one.go#L884

No need to increase the verbosity in this line.

Reply:
understood, I'll leave the PR as it is and wait for additional review/approval